.eslintrc.js

```diff
@@ -46,7 +46,7 @@ module.exports = {
 		"no-void": "error",
 		"wrap-iife": ["error", "inside"],
 		"no-shadow": ["error", { builtinGlobals: true, hoist: "never" }],
-		"no-use-before-define": "error",
+		"no-use-before-define": "off",
```

Comment:
Why remove this?

Reply:
It's because of the new JSON types. There is no way to order them that doesn't use a type that hasn't been defined yet.